### PR TITLE
Eng 10509

### PR DESCRIPTION
**Pull request by hhakimi53**
Pull request for initial commit of docker files. This uses voltdb community edition.

### CODE REVIEW

File: tools/docker/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,62 @@
+FROM ubuntu:14.04
```
Comment (vtkstef) [inline tools/docker/Dockerfile]: is there an LTS version of this?

File: tools/docker/deployment.xml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0"?>
+<deployment>
+    <cluster sitesperhost="2" kfactor="0" />
+    <httpd enabled="true">
+        <jsonapi enabled="true" />
+    </httpd>
+    <!--paths>
```
Comment (vtkstef) [inline tools/docker/deployment.xml]: remove commented stanza

File: tools/docker/Dockerfile (hunk 1 above)
Comment (hhakimi53) [inline tools/docker/Dockerfile]: That's an LTS image. For trusty docker image, there are tags for image named 14.04.5, 14.04, trusty-20160819, trusty and looks like they all point to the same image based on the repo.

File: tools/docker/deployment.xml (hunk 2 above)
Comment (hhakimi53) [inline tools/docker/deployment.xml]: I missed cleaning this one. It was there for testing with different permutations. Thnxs

**Pull request merged**